deploy_preservica_config.tf

```diff
@@ -0,0 +1,80 @@
+locals {
+  preservica_config_bucket_name = "${local.environment}-dr2-preservica-config"
+  preservica_config_lambda_name = "${local.environment}-preservica-config"
+}
+module "preservica_config_bucket" {
+  source      = "git::https://github.com/nationalarchives/da-terraform-modules//s3"
+  bucket_name = local.preservica_config_bucket_name
+  sns_topic_config = {
+    "s3:ObjectCreated:*" = module.preservica_config_sns.sns_arn
+  }
+  kms_key_arn = module.dr2_developer_key.kms_key_arn
+  bucket_policy = templatefile("${path.module}/templates/s3/preservica_config_bucket_policy.json.tpl", {
+    preservica_config_lambda_role_arn = module.preservica_config_lambda.lambda_role_arn
+    bucket_name                       = local.preservica_config_bucket_name
+  })
+  logging_bucket_policy = templatefile("${path.module}/templates/s3/log_bucket_policy.json.tpl", {
+    bucket_name = "${local.preservica_config_bucket_name}-logs", account_id = var.dp_account_number
+  })
+}
+
+module "preservica_config_sns" {
+  source = "git::https://github.com/nationalarchives/da-terraform-modules//sns"
+  sns_policy = templatefile("${path.module}/templates/sns/s3_notifications_policy.json.tpl", {
+    topic_name  = "${local.environment}-preservica-config"
+    bucket_name = "${local.environment}-dr2-preservica-config"
+    account_id  = data.aws_caller_identity.current.account_id
+  })
+  tags = {
+    Name = "Preservica Config SNS"
+  }
+  topic_name  = "${local.environment}-preservica-config"
+  kms_key_arn = module.dr2_developer_key.kms_key_arn
+  sqs_subscriptions = {
+    preservica_config_queue = module.preservica_config_queue.sqs_arn
+  }
+}
+
+module "preservica_config_queue" {
```

Comment:
May require a greater than default max receives as the config may have some decencies on other invocations of this lambda. The default of 3 seems quite low as well.

Reply:
How many do you reckon? 6 seems ok. I'll put a note in to change the default if that's sensible.